modules/redis/src/main/scala/zio/redis/RedisUri.scala

```diff
@@ -26,19 +32,29 @@ object RedisUri {
     val splitting = hostAndPort.split(':')
     val host      = splitting(0)
     val port      = splitting(1).toInt
-    RedisUri(host, port, ssl = false, sni = None)
+    RedisUri(host, port, ssl = false, sni = None, verifyCertificate = true)
```

Comment:
Let's omit the defaults:
```suggestion
    RedisUri(host, port, ssl = false, sni = None)
```